### ISSUE TITLE
Release probes and success-rate datasets on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [KabakaWilliam/llms_know_difficulty#13]
Hi @CoffeeGitta 🤗

Niels here from the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2602.09924.
The paper page lets people discuss about your paper and lets them find artifacts about it (your models, datasets or demo for instance), you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

I saw you've released the code for PIKA, which is great! It'd be awesome to also make the pre-trained linear probes and the generated success-rate datasets (the model performance labels for MATH, GSM8K, AIME, and the E2H-AMC data) available on the 🤗 hub. This would significantly improve the discoverability and reproducibility of your work, allowing others to use your probes for routing right away.

#
